src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-tabs/smb-tabs.component.html

```diff
@@ -0,0 +1,29 @@
+
+<cds-tabs
+  type="contained"
+  followFocus="true"
+  isNavigation="true"
+  cacheActive="true">
+  <cds-tab
+    heading="Clusters"
+    i18n-heading>
+    <cd-smb-cluster-list
+      [clusters]="clusters"
+    ></cd-smb-cluster-list>
+  </cds-tab>
+  <cds-tab
+    heading="Active Directory"
+    i18n-heading>
+    <cd-smb-join-auth-list
+      [joinAuths]="joinAuths"
+    ></cd-smb-join-auth-list>
+  </cds-tab>
+  <cds-tab
+    heading="Standalone"
```

Comment:
for each of these tabs we should add one line help text here, thanks/ Doc team can help here

Reply:
What do you exactly mean by line help text, the `cd-helper` component? How would that work with the tab?

I was trying to set a tooltip for the tab, but could not make it work, it was not showing the content, in the video can be seen very small that pointer of the tooltip is showing when hovering over the tab but nothing else.

[tooltipnotshowing.webm](https://github.com/user-attachments/assets/88d28ebc-ce0e-4baa-9ef3-c1c0f80703cf)


But anyways, since I've updated it to display a whole header for the page for each of the tabs, I think we don't really need the tooltips

Reply:
again taking an example from nvmeof component, maybe this is what @afreen23 was talking about? But I don't have a strong preference on whether the Heading and text is inside the tab content or outside it. I prefer consistency more or less.

![image](https://github.com/user-attachments/assets/d6bf4748-6fac-495d-83eb-09c6acd6c8cb)

Reply:
Yes we can do that way. @Pegonzal lets do same way of if there is a carbon way do in that manner. Thanks

Reply:
We should achieve consistency.
Also @Pegonzal you may check with UX on how to put. What we did with nvme was not something cross verified.
So outside/inside lets clarify with UX team please. Thanks for all gr8 work

Reply:
I talked with UX, I am waiting on them to get their feedback back to me. As per their estimations, this Thursday I should have updates regarding the feedback.

For now, I'm following the same pattern as in NFS, except that I'm udpating the text displayed depending on the active tab

Reply:
The problem here is consistency. I really dont like the fact we are disregarding consistency across product. As far as current implementation is concerned this is not like nvmeof. Moving ahead I would prefer to see consistent carbonized components.

For here, if we choose tab headings inside tab then let it be for all why go away with that ?
And if we choosing here outside tab then have it outside for all tabs and fix other places similarly.

Thanks